github/workflows/test.yml

```diff
@@ -19,6 +19,8 @@ jobs:
         ports:
           - 5432:5432
     steps:
+      - name: Install dependencies
+        run: sudo apt-get update && sudo apt-get install -y libvips
```

Comment:
This makes every run a lot slower for the benefit of projects that use active storage. We don't actually configure active storage out of the box (yet?).

You could consider this pattern (from FRG) instead:

```
        run: sudo apt-get install -y libpq-dev libcurl4-openssl-dev libvips ||
          (sudo apt-get update && sudo apt-get install -y libpq-dev libcurl4-openssl-dev libvips)
```

IE don't run `apt-get update` unless install fails without it.

Reply:
we could do this
```
      - name: Install dependencies
        if: exists('config/storage.yml')
        run: sudo apt-get install -y libvips || (sudo apt-get update && sudo apt-get install -y libvips)
```

Reply:
I think we're always going to have differences between projects. I'm advocating for adding this when required rather than having it in the template. The counter argument could be that we want a good pattern for adding dependencies rather than asking everyone to work it out for themselves.